docs/deployment_patterns/how-to-instantiate-a-data-context-on-an-emr-spark-cluster.md

```diff
@@ -1,5 +1,46 @@
 ---
 title: How to instantiate a Data Context on an EMR Spark cluster
 ---
+import Prerequisites from '../guides/connecting_to_your_data/components/prerequisites.jsx'
+
+This guide will help you instantiate a Data Context on an EMR Spark cluster.
+
+
+The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations [command line interface (CLI)](../guides/miscellaneous/how-to-use-the-great-expectations-cli.md).
+
+
+<Prerequisites>
+
+- [Followed the Getting Started tutorial and have a basic familiarity with the Great Expectations configuration](../tutorials/getting-started/intro.md).
+
+</Prerequisites>
+
+Steps
+-----
+
+1. **Install Great Expectations on your EMR Spark cluster.**
+
+   Copy this code snippet into a cell in your EMR Spark notebook and run it:
+
+    ```python
```

Comment:
Same question as for a previous PR: do we want to un-indent "```python", "```yaml", "```bash", and similar maximally to the left?

Reply:
Resolving per conclusion arrived at this morning about codeblock alignment.